mandatory/3-function-output.js

```diff
@@ -1,16 +1,21 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// Add comments to explain what this function does. You're meant to use Google! 
+/* Math.random returns a pseudo-random number between 0 - less than 1 (by default) 
```

Comment:
This definition is kinda generic. You should have tailored it to what the underlying method do

Reply:
Will look up for a new more defining definition.

Reply:
Done, thanks for the feedback.